### DISCUSSION TITLE
Online-only meetings not publishing to the Meeting App (Location/geocoding issue)

### DISCUSSION THREAD

**Discussion by Catherine-Area80** [code4recovery/12-step-meeting-list#1391] [Q&A]
**Website** https://aamanitoba.org/meetings

**Issue** We're experiencing issues with our active online-only meetings not displaying on the Meeting App. The issue seems to be linked with the Location and Map fields. The location for some of the online-only meetings includes the geocoding; these seem to get to the Meeting App. However, the location for other online-only meetings does not include the geocoding; these meetings are missing on the Meeting App. 

**Versions** TSML - Version 3.14.32; Wordpress - Version 6.3.3

TIA!

**Comment by SIA-WEBSUP** [comment]
The plugin allows online meetings with no physical address.

However, for online meetings to be listed by the meeting guide app, they must have an 'approximate address', which usually means a city/town, state and maybe a zip code.

So if you'd like an online meeting to show up in the meeting guide app, give it an 'approximate address'

Here are a couple of other discussion threads addressing ( pun intended ) this same issue:

- https://github.com/code4recovery/12-step-meeting-list/discussions/1316

- https://github.com/code4recovery/12-step-meeting-list/discussions/1251

**Comment by SIA-WEBSUP** [comment]
Hmm... perhaps you know this already but are instead reporting that some of your online meetings ALREADY have 'approximate addresses' but are not being geocoded correctly?

**Reply by Catherine-Area80** [reply]
Thanks!! I think so. We have two Location options showing up under the same 'name'—**online --- Manitoba**. One option populates the Address with an 'approximate location' (Manitoba, Canada) but is not being geocoded correctly. The second option, same name—**online --- Manitoba**, populates a more detailed 'approximate address' (Winnipeg, MB, Canada) and does geocode. Meetings with the latter seem to be posting to the Meeting App just fine. I don't know what's prompted/created these two location options or the change to the online meetings whose Location details was changed. Last week we ran into another issue with the location of online-only meetings that caused a whack of meetings to move from an active/published state to a draft state. We manually updated the Location and things were back to normal (on our website and on the app). We haven't made any manual updates to the meetings currently affected.

**Comment by SIA-WEBSUP** [comment]
I don't think the locationName matters nearly as much as the locationAddress. If you know that Winnipeg, MB, Cananda geocodes correctly, why not use that for the approximate address for all meetings based out of Manitoba?

Alternatively, I think simply Manitoba, Canada also geocodes... but it will gecode to the 'geographic center' of Manitoba, which literally in Lake Winnipeg, close to the northern shore.

![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/0a388ef0-9411-4a93-8b55-3b11db88ee8b)

**Reply by SIA-WEBSUP** [reply]
![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/e732fe41-2178-4a4e-b118-ba4c6162b810)

**Comment by SIA-WEBSUP** [comment]
The meeting guide app is really meant to find meetings geographically close to where you are located ( or wherever you set the 'current location' to ). 

So online meetings in the meeting guide app are a wee confusing since they must be 'based' from some physical location (i.e. have an approximate address). This makes good enough sense when you're dealing with a densely populated area... and perhaps you do want to find an online group that has folks near you.

But for a very large, sparsely populated areas... this makes less sense.  

I'm guessing that what you probably want is some consistency in the case of Manitoba. Perhaps it would be good to have all ONLINE ONLY meetings based ANYWHERE in Manitoba to have an approximate address of 'Manitoba, Canada' which as I already mentioned appears to be in the middle of Lake Winnipeg near the northern end, but no where near the city of Winnipeg.

But if someone did want to find online meetings based ANYWHERE in Manitoba, they could change their 'current location' in the meeting guide app to Manitoba, Cananda and find them all.

**Reply by Catherine-Area80** [reply]
Thank you!! I didn't realize about the geocoding lol but it makes sense now. The meetings seem to be displaying as usual now so not sure if our Webmaster fixed or things sorted themselves out somehow but thanks so much for your help and the detailed explanation. It is much appreciated.

**Reply by SIA-WEBSUP** [reply]
Glad to hear it!! Happy to help. Thank you for allowing me to be of service.
